var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -183,3 +183,8 @@ def cmake_args(self):
                 args.append("-DENABLE_STATIC_LIBS=NO")
 
         return args
+
+    def test_ptls(self):
+      parseThat = which(join_path(self.prefix.bin, "parseThat"))
```

Comment:
Optional. You don't have to use `join_path` here to add `parseThat` to the path.

```suggestion
      parseThat = which(self.prefix.bin.parseThat)
```

Reply:
Oh, I didn't realize that the path objects did that. Thanks for the heads-up!